requirements.txt

```diff
@@ -1 +1,2 @@
 COMPAS >=1.0
+opensees>=0.1.0
```

Comment:
this should indeed be an optional dependency since it is not compatible with Rhino 7